src/js/modules/wallet/controllers/settings/settings.ctrl.js

```diff
@@ -4,702 +4,5 @@
-                                                                        ;
-                                                                } else if (error) {
-                                                                    throw new Error('MSG_BAD_LOGIN');
-                                                                } else {
-                                                                    throw new Error('MSG_BAD_NETWORK');
-                                                                }
-                                                            }
-                                                        )
-                                                        ;
-                                                };
-
-                                                return passwordChange();
-                                            });
-                                    });
-                                });
-                            });
-                    })
-            })
-                .catch(function(err) {
-                    if (err && err.message) {
-                        $scope.alert($translate.instant('CHANGE_PASSWORD'), err.message);
-                    }
-                })
-        }
-
-        $scope.sweepCoins = function () {
-            $modal.open({
-                controller: "SweepCoinsModalController",
-                templateUrl: "js/modules/wallet/controllers/sweep-coins-modal/sweep-coins-modal.tpl.html",
-                size: 'md'
-            });
-        };
-
-        $scope.addProtocolHandler = function() {
-            // Prompt user about this feature
-            return NotificationsService.promptBitcoinURIHandler().result.then(function() {
-                try {
-                    $log.debug('Trying to register bitcoin URI scheme');
-                    navigator.registerProtocolHandler(
-                        'bitcoin',
-                        CONFIG.WALLET_URL + '/#/wallet/handleURI/%s',
-                        'BTC.com Bitcoin Wallet'
-                    );
-                } catch (e) {
-                    $log.error('Couldn\'t register bitcoin: URL scheme', e, e.message);
-
-                    if (e.name === "SecurityError") {
-                        return dialogService.alert(
-                            $translate.instant('ERROR_TITLE_2'),
-                            $translate.instant('BROWSER_SECURITY_ERROR'),
-                            $translate.instant('OK')
-                        ).result;
-                    }
-                }
-            });
-        };
-
-        $scope.$on('$destroy', function() {
-            // Remove existing listeners
-            if(listenerFormSettings) {
-                listenerFormSettings();
-            }
-
-            if(listenerEnabled2faToggle) {
-                listenerEnabled2faToggle()
-            }
-        });
-    }
-
+    function SettingsCtrl() { }
```

Comment:
@romaroid when controller is empty we can just omit it entirely?

Reply:
removed the controller completely, as `app.wallet.settings` is now an abstract state.